lib/src/components/NxGlobalSidebar/__tests__/NxGlobalSidebarFooter.test.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
+
+import NxGlobalSidebarFooter, { NxGlobalSidebarFooterProps as Props }
+  from '../NxGlobalSidebarFooter';
+
+import NxTextLink from '../../NxTextLink/NxTextLink';
+
+describe('NxGlobalSidebarFooter', function() {
+  const minimalProps: Props = {
+        supportText: 'Support for RSC',
+        supportLink: '#supporturl',
+        releaseText: 'React Shared Components',
+        releaseNumber: '3.1.4',
+        productTagLine: 'Powered by PLAID VILLAIN',
+        showCreatedBy: true
+      },
+      getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxGlobalSidebarFooter, minimalProps);
+
+  it('renders an <footer> with nx-global-sidebar__footer class', function() {
+    expect(getShallowComponent()).toMatchSelector('footer.nx-global-sidebar__footer');
+  });
+
```

Comment:
Needs tests for custom class names and attributes.

Reply:
> Needs tests for custom class names and attributes.

Here: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/dcb088dbde236bda5ab9f146b439c8b4beb1d501

Reply:
> Here: dcb088d

Still need a test for arbitrary attributes.  For example https://github.com/sonatype/sonatype-react-shared-components/blob/master/lib/src/components/NxColorPicker/__tests__/NxColorPicker.test.tsx#L30

Reply:
> > Here: [dcb088d](https://github.com/sonatype/sonatype-react-shared-components/commit/dcb088dbde236bda5ab9f146b439c8b4beb1d501)
> 
> Still need a test for arbitrary attributes. For example https://github.com/sonatype/sonatype-react-shared-components/blob/master/lib/src/components/NxColorPicker/__tests__/NxColorPicker.test.tsx#L30

Here it is: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/fe8b4bdc987e12295a8a53ce6f74632864160d17